plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/style.scss

```diff
@@ -25,22 +29,11 @@
 	}
 
 	&:has(.wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill-input:focus) {
-		outline: 1px solid var(--wp--preset--color--accent-4);
+		outline: 1px solid currentColor;
 		outline-offset: 1px;
 	}
 
-	&:has(.wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill-input:checked) {
-		&:hover {
-			background-color: color-mix(
-				in srgb,
-				var(--wp--preset--color--contrast) 85%,
-				transparent
-			);
-		}
-	}
-
 	&:has(.wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill-input:disabled) {
-		border-color: var(--wc-subtext);
 		color: var(--wc-subtext);
```

Comment:
I know it wasn't mentioned in the original issue, but given that we are improving this, it might be a good opportunity to do it. :slightly_smiling_face: 

`var(--wc-subtext)` doesn't work very well in some color combinations, and it might be unavailable in some themes. I wonder if we should replace it with the border color, how does it sound?

```suggestion
		color: color-mix(in srgb,currentColor 40%,transparent);
```

Before | After
--- | ---
![imatge](https://github.com/user-attachments/assets/0f03f944-5024-41cb-b580-4588a75d2359) | ![imatge](https://github.com/user-attachments/assets/41a87ae9-539e-46a0-933f-bc46956f4a6c)